akka-docs/rst/java/http/routing-dsl/directives/header-directives/optionalHeaderValue.rst

```diff
@@ -6,7 +6,7 @@ optionalHeaderValue
 Description
 -----------
 Traverses the list of request headers with the specified function and extracts the first value the function returns as
-``Some(value)``.
+``Optional[value]``.
```

Comment:
Oh, I see. Looked "like scala to me" and thought it's an omission. If you think Java people will like this let's keep it